src/main/interlok/config/jetty.xml

```diff
@@ -221,7 +221,7 @@
                   -->
                 </New>
               </Set>
-              <Set name="tempDir">./fs/temp-files</Set>
+              <Set name="tempDir">/tmp</Set>
```

Comment:
i.e. you can just remove line 224 completely

Reply:
Happy with that option if it improve interoperability. I'll tweak and re-request the pull.